test/browser/features/DVLADrivingLicence.feature

```diff
@@ -19,183 +19,198 @@ Feature: DVLA Driving licence CRI Error Validations
       |DrivingLicenceSubjectHappyPeter|E20A              |
 
-  @mock-api:dvla-invalidPostcode @validation-regression @build @staging
+  @mock-api:dl-success @validation-regression @build @staging
   Scenario Outline: DVLA Driving Licence - No Postcode in the Postcode field error validation
     Given User enters DVLA data as a <DrivingLicenceSubject>
     And User re-enters postcode as <InvalidPostcode>
     When User clicks on continue
     Then I see the postcode error in summary as Enter your postcode
     And I see the postcode error in field as Enter your postcode
+    And I check the page Title Error: Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
     Examples:
       |DrivingLicenceSubject          |InvalidPostcode   |
       |DrivingLicenceSubjectHappyPeter|                  |
 
-  @mock-api:dvla-invalidPostcode @validation-regression @build @staging
+  @mock-api:dl-success @validation-regression @build @staging
   Scenario Outline: DVLA Driving Licence International Postcode error validation
     Given User enters DVLA data as a <DrivingLicenceSubject>
     And User re-enters postcode as <InvalidPostcode>
     When User clicks on continue
     Then I see the postcode error in summary as Enter a UK postcode
     And I see the postcode error in field as Enter a UK postcode
+    And I check the page Title Error: Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
     Examples:
       |DrivingLicenceSubject          |InvalidPostcode   |
       |DrivingLicenceSubjectHappyPeter|CA 95128          |
 
 ##### PostcodeWithSpecialChar #####
-  @mock-api:dvla-invalidPostcode @validation-regression @build @staging
+  @mock-api:dl-success @validation-regression @build @staging
   Scenario Outline: DVLA Driving Licence Postcode with special characters error validation
     Given User enters DVLA data as a <DrivingLicenceSubject>
     And User re-enters postcode as <InvalidPostcode>
     When User clicks on continue
     Then I see the postcode error in summary as Your postcode should only include numbers and letters
     And I see the postcode error in field as Your postcode should only include numbers and letters
+    And I check the page Title Error: Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
     Examples:
       |DrivingLicenceSubject          |InvalidPostcode   |
       |DrivingLicenceSubjectHappyPeter|NW* ^%G           |
 
 ###### PostcodeWithNumericChar, PostcodeWithAlphaChar #####
-  @mock-api:dvla-invalidPostcode @validation-regression @build @staging
-  Scenario Outline: DVLA Driving Licence Postcode with numeric characters or alpha characters error validation
+  @mock-api:dl-success @validation-regression @build @staging
+  Scenario Outline: DVLA Driving Licence Postcode with numeric characters or Postcode with numeric characters error validation
```

Comment:
Hi @MathumithaVe Could you please revert this line to what it was before i.e. Change this line 'Scenario Outline: DVLA Driving Licence Postcode with numeric characters or Postcode with numeric characters error validation' to 'Scenario Outline: DVLA Driving Licence Postcode with numeric characters or alpha characters error validation'